src/tests/ftest/deployment/ior_per_rank.yaml

```diff
@@ -0,0 +1,61 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+timeout: 1500
+server_config:
+    name: daos_server
+    engines_per_host: 2
+    engines:
+    0:
+      pinned_numa_node: 0
+      nr_xs_helpers: 1
+      fabric_iface: ib0
+      fabric_iface_port: 31317
+      log_file: daos_server0.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem0"]
+          scm_mount: /mnt/daos0
+        1:
+          class: nvme
+          bdev_list: ["aaaa:aa:aa.a"]
+    1:
+      pinned_numa_node: 1
+      nr_xs_helpers: 1
+      fabric_iface: ib1
+      fabric_iface_port: 31417
+      log_file: daos_server1.log
+      storage:
+        0:
+          class: dcpm
+          scm_list: ["/dev/pmem1"]
+          scm_mount: /mnt/daos1
+        1:
+          class: nvme
+          bdev_list: ["bbbb:bb:bb.b"]
+pool:
+  mode: 146
+  name: daos_server
+  size: 95%
+  control_method: dmg
+  properties: ec_cell_sz:128KiB
+container:
+    type: POSIX
+    properties: cksum:crc16,cksum_size:16384,srv_cksum:on
+    control_method: daos
+ior:
+    client_processes:
+          ppn: 16
+    test_file: testFile
+    repetitions: 1
+    flags: "-w -r -R -C -e -g -D 30 -G 27 -k -Q 1 -v"
+    dfs_oclass: 'SX'
+    transfer_sizes:
+        - 1M
+        - 256B
+    block_size: 150G
+    write_x: 0.1 # Max 10% performance difference.
+    read_x: 0.15  # Max 15% performance difference.
+    expected_bw: 22500    # For transfer size 1M
+    expected_iops: 600000       # For transfer size 256B
```

Comment:
In CI, I'd recommend making these both 15%. Similar tests intermittently fail with 10% margins.

Reply:
Done